ethcore/src/evm/interpreter/mod.rs

```diff
@@ -292,14 +292,21 @@ impl<Cost: CostType> Interpreter<Cost> {
 					}
 				};
 			},
-			instructions::CALL | instructions::CALLCODE | instructions::DELEGATECALL => {
+			instructions::CALL | instructions::CALLCODE | instructions::DELEGATECALL | instructions::STATICCALL => {
 				assert!(ext.schedule().call_value_transfer_gas > ext.schedule().call_stipend, "overflow possible");
+				if instruction == instructions::STATICCALL && !ext.schedule().static_call {
+					// STATICCALL is not enabled, yet called
+					return Err(evm::Error::BadInstruction {
+						instruction: instruction
+					});
+				}
+
```

Comment:
after looking at Arkadiy eip86, I guess this should be under `verify_instruction`